validator/client/registration.go

```diff
@@ -29,17 +31,27 @@ func SubmitValidatorRegistrations(
 		return nil
 	}
 
-	if _, err := validatorClient.SubmitValidatorRegistrations(ctx, &ethpb.SignedValidatorRegistrationsV1{
-		Messages: signedRegs,
-	}); err != nil {
-		if strings.Contains(err.Error(), builder.ErrNoBuilder.Error()) {
-			log.Warnln("Beacon node does not utilize a custom builder via the --http-mev-relay flag. Validator registration skipped.")
-			return nil
+	chunks := chunkSignedValidatorRegistrationV1(signedRegs, validatorRegsBatchSize)
+	var lastErr error
+
+	for _, chunk := range chunks {
+		innerSignerRegs := ethpb.SignedValidatorRegistrationsV1{
+			Messages: chunk,
 		}
-		return errors.Wrap(err, "could not submit signed registrations to beacon node")
+
+		if _, err := validatorClient.SubmitValidatorRegistrations(ctx, &innerSignerRegs); err != nil {
+			if strings.Contains(err.Error(), builder.ErrNoBuilder.Error()) {
+				log.Warnln("Beacon node does not utilize a custom builder via the --http-mev-relay flag. Validator registration skipped.")
+			}
+
+			log.WithError(err).Warn("Could not submit signed registrations to beacon node")
+			lastErr = errors.Wrap(err, "could not submit signed registrations to beacon node")
+		}
+
+		log.Infoln("Submitted builder validator registration settings for custom builders")
```

Comment:
Also this gets logged exactly the same if there are or there aren't errors, perhaps it's simpler to just log the error if there is a `lastErr` and log this otherwise.